codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/GoStdlibTypes.java

```diff
@@ -35,6 +35,7 @@ public static final class Context {
 
     public static final class Time {
         public static final Symbol Time = SmithyGoDependency.TIME.pointableSymbol("Time");
+        public static final Symbol Duration = SmithyGoDependency.TIME.valueSymbol("Duration");
```

Comment:
FWIW we've kind of stopped doing this because there are just too many -- instead we just started calling these directly, and we added some helpers so you can do stuff like

`SmithyGoDependency.TIME.struct("Time")`

This is fine though

Reply:
Got it, nah I rather do it The Right Way ™

Will close the PR